src/main/scala/atom/Atom.scala

```diff
@@ -187,11 +187,27 @@ trait NotificationManager extends js.Object {
   def getNotifications(): js.Array[Notification] = js.native
 }
 
+class MenuOptions(
```

Comment:
I see that this is called `MenuOptions` in the TS facade, but I think a better name would be `MenuItem` (from the docs):

> `items` An `Array` of **menu item** `Objects` containing the keys